main.go

```diff
@@ -49,9 +49,10 @@ var (
 	dumpFiles = dump.Arg("file", "Certificate file to dump (or stdin if not specified).").ExistingFiles()
 	dumpType  = dump.Flag("format", "Format of given input (heuristic guess if not specified).").String()
 
-	connect     = app.Command("connect", "Connect to a server and print its certificate.")
-	connectTo   = connect.Arg("server:port", "Hostname or IP to connect to.").String()
-	connectName = connect.Flag("name", "Override the server name used for SNI.").String()
+	connect       = app.Command("connect", "Connect to a server and print its certificate.")
+	connectTo     = connect.Arg("server:port", "Hostname or IP to connect to.").String()
+	connectName   = connect.Flag("name", "Override the server name used for SNI.").String()
+	connectCaPath = connect.Flag("ca", "Path to CA bundle (system default if unspecified).").ExistingFile()
```

Comment:
It's a system-specific value/function, built into the Go runtime. For example on macOS it's the keychain, whereas on Linux it's a distro-specific file path.